backend/src/main/java/heartbeat/client/dto/pipeline/buildkite/BuildKiteBuildInfo.java

```diff
@@ -34,7 +34,7 @@ public DeployInfo mapToDeployInfo(String step, String state, String startTime, S
 			.filter(item -> Objects.equals(item.getName(), step) && Objects.equals(state, item.getState()))
 			.filter(item -> {
 				Instant time = Instant.parse(item.getFinishedAt());
-				return time.isAfter(startDate) && time.isBefore(endDate);
+				return time.compareTo(startDate) >= 0 && time.compareTo(endDate) <= 0;
```

Comment:
Why covert to this? isAfter and isBefore is better than yours

Reply:
isAfter and isBefore do not include `=`, their internal implementation is `compareTo(otherInstant) >0 / < 0`.

Reply:
extract TimeUtil Interface